### ISSUE TITLE
Can't reset default config with false from stylelint-config-standard

### ISSUE THREAD

**Issue by alexander-schranz** [stylelint/stylelint#2556]
Rest stylelint default config not working with false.

> Which rule, if any, is this issue related to?

any e.g. `"no-empty-source": false`

> What CSS is needed to reproduce this issue?

```css
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "no-empty-source": false
  }
}
```

> Which version of stylelint are you using?

e.g. `6.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

empty css

> What did you expect to happen?

success lint.

> What actually happened (e.g. what warnings or errors you are getting)?

`Unexpected option value "false" for rule "no-empty-source"`

*Workaround*:

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "no-empty-source": null
  }
}
```

Think `false` should behave like `null`.

**Comment by hudochenkov** [comment]
Thank you for using a template!

It's intended behaviour, `no-*` rules could have only `true` option. `false` make not sense to them, because if it's not forbidden, that it's allowed. If it's allowed, then rule should not work. `null` is for disabling rules.

**Comment by hudochenkov** [comment]
@alexander-schranz BTW, why are you using so old version?

**Comment by alexander-schranz** [comment]
@hudochenkov but when you extends you want to overwrite a `no-*` rule which you need then set to `null` which feels hacky for me.

oh sorry thought the issue template has always the newest version in it. did check its `7.10.1`.
